docs/getting_started.md

```diff
@@ -144,3 +144,20 @@ The `field` and `primary` macros have a comment option that will specify the doc
 `field age : Int32, comment: "# Number of seconds since the post was posted"`
 
 See the [Docs folder](./) for additional information.
+
+### Third-Party Annotations
+
+Annotations can be a powerful method of adding property specific features with minimal amounts of code.  Since Granite utilizes the `property` keyword for its fields, annotations are able to be applied easily.  This is accomplished by using the `annotations` option on a field, similar to the `comment` option above.  It is used in the form of `annotations: ["annotation1", "annotation2"]`, for example:
+
+```Crystal
+class Foo < Granite::Base
+  adapter mysql
+  table_name foos
+
+  field name : String
+  field password : String, annotations: ["@[Foo::Options(option1: true)]", "@[Bar::Settings(other_option: 7)]"]
```

Comment:
@Blacksmoke16 Is it possible to apply the annotations to the field?
```
@[Foo::Options(option1: true)]
field password : String
```

Reply:
I don't think so unfortunately.  I'll try it again but i think it messes with it since its a macro and not the actual property.

Reply:
bummer

Reply:
Kinda late, but I revisited this.  It _is_ possible to apply annotations to properties created via a macro.  However it will _not_ work with Granite since our `field` macro just adds data to a mutable constant and not actually defines the property.